### PR TITLE
Adding initial troubleshooting guide

### PR DESCRIPTION
**Pull request by ewolinetz**

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Cluster Logging Operator Troubleshooting
```
Comment (jcantrill) [inline TROUBLESHOOTING.md:1]: Can we start a 'docs' folder

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+## Known issues
+
+### After Upgrading CLO and then EO, CLO doesn't know about `kibana` type
+Part of the work done for between CLO & EO 4.4 -> 4.5 was to change which operator was responsible for Kibana. This included creating a new CRD for Kibana that the EO watches for.
```
Comment (jcantrill) [inline TROUBLESHOOTING.md:6]: The responsibility of deploying Kibana was moved to the EO from CLO as part of the 4.5 feature work. ....

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+If CLO is upgraded before EO is, then it will try to create a Kibana CR but the Kibana CRD has not yet been created. This will cause CLO to error out with messages indicating it does not know about the type "Kibana".
```
Comment (jcantrill) [inline TROUBLESHOOTING.md:8]: ...upgraded before EO, it will try to create a ...

File: TROUBLESHOOTING.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+If this happens, ensure that EO has been updated to at least 4.5 as well so that the Kibana CRD will be created and then delete your running CLO pod. It should restart without the prior error messages and you should see a new Kibana instance be rolled out (it will be managed by EO instead).
```
Comment (jcantrill) [inline TROUBLESHOOTING.md:10]: It will restart.... and you will see...instance roll out

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (ewolinetz) [reply TROUBLESHOOTING.md:1]: we already have one, moving this to there

**Pull request merged**